services-custom/dynamodb-enhanced/src/main/java/software/amazon/awssdk/enhanced/dynamodb/internal/EnhancedClientUtils.java

```diff
@@ -54,6 +54,7 @@ public static String cleanAttributeName(String key) {
                 || chars[i] == '.'
                 || chars[i] == '-'
                 || chars[i] == '#'
+                || chars[i] == '+'
```

Comment:
Nit: Can also consider a `Set<Character>`.

Reply:
For storing the special characters, you mean?
I'll change the next time a special character needs to be added :)